src/main/java/org/tailormap/api/controller/admin/TaskAdminController.java

```diff
@@ -136,6 +136,7 @@ public ResponseEntity<Object> list(@RequestParam(required = false) String type)
                         .put(
                             Task.DESCRIPTION_KEY,
                             jobDetail.getJobDataMap().getString(Task.DESCRIPTION_KEY))
+                        .put("lastResult", jobDetail.getJobDataMap().getString("lastResult"))
```

Comment:
You should probably add a key `String LASTRESULT_KEY = "lastResult";` in the Task interface: 
https://github.com/Tailormap/tailormap-api/blob/cde9ed11f5bb61f55ab490fe4a96b63d8fa5bd93/src/main/java/org/tailormap/api/scheduling/Task.java#L8-L16

and use that key in any place that now use a String: https://github.com/search?q=repo%3ATailormap%2Ftailormap-api+lastResult&type=code